src/Tools/PrepareTests/MinimizeUtil.cs

```diff
@@ -64,10 +68,34 @@ void initialWalk()
                     else
                     {
                         var destFilePath = Path.Combine(currentOutputDirectory, fileName);
-                        CreateHardLink(destFilePath, sourceFilePath, IntPtr.Zero);
+                        CreateHardLink(destFilePath, sourceFilePath);
                     }
                 }
             }
+
+            // https://github.com/dotnet/roslyn/issues/49486
+            // we should avoid copying the files under Resources.
+            var individualFiles = new[]
+            {
+                "./global.json",
+                "src/Workspaces/MSBuildTest/Resources/.editorconfig",
+                "src/Workspaces/MSBuildTest/Resources/global.json",
+                "src/Workspaces/MSBuildTest/Resources/Directory.Build.props",
+                "src/Workspaces/MSBuildTest/Resources/Directory.Build.targets",
+                "src/Workspaces/MSBuildTest/Resources/Directory.Build.rsp",
+                "src/Workspaces/MSBuildTest/Resources/NuGet.Config",
+            };
```

Comment:
Why did you keep them here vs. putting them under the eng directory?

Reply:
Unit tests within `src/Workspaces/MSBuildTest` depend on being able to embed these resources. Moving them seemed to introduce more pain than just leaving them here and including them in the test payload to make `eng/build.ps1` happy.

Reply:
Hmm, thought I had mostly fixed up the Unit Tests problems in the Clone2 PR. Must have missed a few. Can clean this up later though.

Reply:
I am hoping that with @jasonmalinowski's guidance, it will be fairly straightforward to fix #49486 and remove this concern from `eng/build.ps1` entirely.